opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "alertingDashboards",
-  "version": "1.3.2.0",
+  "version": "1.3.3.0",
   "opensearchDashboardsVersion": "1.3.2",
```

Comment:
Are we not changing the `opensearchDashboardsVersion` to 1.3.3?

Reply:
Thanks @lezzago, just pushed in my latest commit.